From b666f74aed03d1b510c7d3da2cc1be59c73dd317 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Mon, 15 Apr 2019 05:21:48 -0700
Subject: [PATCH] Set BUILD_TESTING=OFF to avoid issue with test installation.

---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index 9c9604d9..ff4db1ba 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -27,6 +27,7 @@ override_dh_auto_configure:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
+		-DBUILD_TESTING=OFF \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DAMENT_PREFIX_PATH="@(InstallationPrefix)"
 
-- 
2.31.1

